apps/docs/content/nginx/how-to/env-variables.mdx

```diff
@@ -194,3 +194,45 @@ You can use runtime env variables in the build environment using the `RUNTIME_`
 If you create a secret env variable and a basic runtime env variable with the same key, Zerops saves the basic runtime env variable from your zerops.yml and ignores the secret env variable.
 
 If you create a basic build env variable and a runtime env variable with the same key, Zerops saves both because the build and runtime environments have separate sets of env variables.
+
+## Prerender.io Support
+
+Zerops provides built-in prerender.io support for SEO optimization. Configure it using these environment variables:
+
+<table className="w-fit my-1.5">
+  <thead>
+    <tr className="text-center rounded-lg">
+      <th className="w-fit"><strong>Variable</strong></th>
+      <th className="w-fit"><strong>Required</strong></th>
+      <th className="w-fit"><strong>Description</strong></th>
+      <th className="w-fit"><strong>Default</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr className="text-center">
+      <td className="w-fit">PRERENDER_TOKEN</td>
+      <td className="w-fit">Yes</td>
+      <td className="w-fit">Your prerender.io service token (set as secret)</td>
```

Comment:
you can skip the `(set as secret)`, it's in the tip below :+1:

Reply:
Done!